Kconfig

```diff
@@ -34,6 +34,21 @@ config CLOUD_BACKEND
 	string
 	default "AWS_IOT"
 
+config USE_CUSTOM_MQTT_CLIENT_ID
+	bool "Use custom MQTT client ID"
+	help
+	  By default the device's IMEI is used as the client ID
+	  for the MQTT connection. This allows to use a fixed
+	  value instead. This is mostly useful when providing
+	  firmware builds to continuous integration tests on
+	  real hardware.
+	  Note: you must only run this firmware build on one
+	  device at a time.
+
+config MQTT_CLIENT_ID
+	depends on CUSTOM_MQTT_CLIENT_ID
+	string	"Configures the custom MQTT client ID"
+
```

Comment:
Maybe provide a default client id here. So that we give the user some sense of what is to be set here. Also, the default value set by kconfig if no is provided ("") makes the ```aws_topics_populate()``` function crash.

Reply:
Done.